src/Middleware/RequestQueue/ref/Microsoft.AspNetCore.RequestQueue.netcoreapp3.0.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Builder
+{
+    public static partial class HstsBuilderExtensions
```

Comment:
It's wierd that this stuff is showing up for this assembly no?

Reply:
It think this was copied. I'll work with @DylanDmitri to regen.